### PR TITLE
Refactor converting to Celsius

### PR DESCRIPTION
**Pull request by dawidkostrzewa**

### CODE REVIEW

File: weatherApp.js [hunk 1]
```diff
@@ -16,6 +16,8 @@ async function showWeather(city) {
 
 showWeather('Białystok');
 
+const KELVIV_FACTOR = 273.15;
```
Comment (Naygel22) [inline weatherApp.js:19]: dobre?

File: weatherApp.js [hunk 2]
```diff
@@ -78,10 +80,15 @@ function createElements(data) {
     weatherImg.src = "images/snow.png";
   }
 
+
+  function convertToCelsius(temp) {
+    return Math.ceil(temp - KELVIV_FACTOR)
+  }
```
Comment (Naygel22) [inline weatherApp.js:86]: good?

File: weatherApp.js [hunk 3]
```diff
@@ -78,10 +80,15 @@ function createElements(data) {
...
+
   const temperature = document.createElement('div');
   temperature.classList.add('temperature');
   app.appendChild(temperature);
-  temperature.textContent = `${Math.ceil(data.main.temp - 273.15)}°C`;
```
Comment (Naygel22) [inline weatherApp.js:84]: you sure??

**Review by Naygel22 — APPROVED** [review]

File: weatherApp.js (hunk 1 above)
Comment (dawidkostrzewa) [reply weatherApp.js:19]: yes

**Pull request merged**